src/lib/AccordionItem.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+  import { slide } from "svelte/transition";
+  // TODO Make this more generic by taking some of these as props too or forwarding events
+  import {
+    formOpen,
+    mapHover,
+    openFromSidebar,
+    sidebarHover,
+  } from "../stores.js";
+
+  export let id: number;
+  export let label: string;
+
+  $: isOpen = $formOpen == id;
+  const toggle = () => {
+    formOpen.update((current) => {
+      if (current == id) {
+        return null;
+      } else {
+        return id;
+      }
+    });
+    if ($formOpen == id) {
+      // Always set this to null first, to force subscribers to see the update.
+      // It's possible to open something from the sidebar, close it (by
+      // clicking on the map or using the sidebar), then reopen the same thing.
+      openFromSidebar.set(null);
+      openFromSidebar.set(id);
+    }
+  };
+
+  // TODO Design choice: While a form is open and we hover on something else in
```

Comment:
The form is visibly open, so we don't need to underline its label. This differs from the hover effect being fixed on the map while a form is open -- we don't want 2 styles for "currently active object" and "possibly different object we're hovering on". Unless someone says otherwise. :)